ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -826,7 +838,7 @@ export default class ConfirmTransactionBase extends Component {
       toAddress,
       showCustodianDeepLink,
       clearConfirmTransaction,
-      isSmartTransactionsEnabled,
+      isSmartTransaction,
```

Comment:
Given the `isSmartTransaction` property was already in the `PropTypes` and it feels a more accurate name given the many conditions included in addition to enablement, I've removed the duplicate `isSmartTransactionsEnabled`.

Reply:
Thanks